now.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "flamingo",
+    "version": 2,
```

Comment:
That's the `now` version, not Flamingo's one right?

Reply:
Yup https://zeit.co/docs/v2/advanced/configuration#version